dashboard/app/models/concerns/curriculum/course_types.rb

```diff
@@ -15,7 +15,8 @@ module Curriculum::CourseTypes
     validates :participant_audience, acceptance: {accept: Curriculum::SharedCourseConstants::PARTICIPANT_AUDIENCE.to_h.values, message: 'must be facilitator, teacher, or student'}
 
     validate :cannot_have_same_audiences
-    validate :must_have_same_course_type_as_family
+    # (Dani) Turning off this validation while I move over K5 self paced pl to the right audience
+    # validate :must_have_same_course_type_as_family
```

Comment:
this seems like it will work for production, however it could also be worth prompting developers to reseed after this change goes in and before you turn the validation back on so that we don't break their local environments.

Reply:
Good reminder. I remember that had been an issue last time and was planning to give developers a couple heads up before turning it back on